internal/validators/status/validator.go

```diff
@@ -135,11 +135,20 @@ func (sv *statusValidator) listGhaStatuses(ctx context.Context) ([]*ghaStatus, e
 		return nil, err
 	}
 
+	// Because multiple jobs with the same name may exist when jobs are created dynamically by third-party tools, etc.,
+	// only the latest job should be managed.
+	currentJobs := make(map[string]bool)
```

Comment:
Why is this a bool value? I think it would be clearer that the value won't be used and this is a mere notation for a set, by using `struct{}` as the value?

Reply:
I agree with your opinion. I will fix it.

Reply:
I have fixed: [apply suggestion](https://github.com/upsidr/merge-gatekeeper/pull/28/commits/35597d2ca3038fb66c9059f653fb78e8a2388d7b)